<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPServer</class>
 <widget class="QWidget" name="TCPServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>发言窗口</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTextEdit" name="edtRecv_2">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>250</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>考场设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>1）考场IP地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtIP">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>2）考场端口号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtPort">
            <property name="text">
             <string>12345</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="text">
             <string>建立考场</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnect_2">
            <property name="text">
             <string>开始考试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnect_3">
            <property name="text">
             <string>结束考试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClear">
            <property name="text">
             <string>清除发言</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClear_2">
            <property name="text">
             <string>禁言</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbxConnection">
            <item>
             <property name="text">
              <string>全部考生</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbHexSend">
            <property name="text">
             <string>Hex发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbHexShow">
            <property name="text">
             <string>Hex显示</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>610</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>广播窗口</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTextEdit" name="edtSend_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>75</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ServerSendInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
